notebooks/galaxy_galaxy_lensing_tutorial.ipynb

```diff
@@ -2,8 +2,13 @@
  "cells": [
```

Comment:
<p>I don't think you need to push this notebook as no changes have been made</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/sim-pipeline/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>